source/collection-management/timeseries-collection.json

```diff
@@ -0,0 +1,110 @@
+{
+  "description": "timeseries-collection",
+  "schemaVersion": "1.0",
+  "runOnRequirements": [
+    {
+      "minServerVersion": "4.9",
+      "serverParameters": {
+        "featureFlagTimeseriesCollection": {
+          "value": true,
+          "version": "5.0"
+        }
```

Comment:
This `featureFlag` seems to be required to create time-series collections. Unfortunately, this means these tests can only run against a config started with `--setParameter featureFlagTimeseriesCollection=true`, which may mean Evergreen changes for some drivers.

Reply:
I know the feature flag for dots and dollars ([DRIVERS-1237](https://jira.mongodb.org/browse/DRIVERS-1237)) was recently removed. Is it likely this feature flag will also be removed before RC0 is tagged next week? Or is this feature likely to sit behind a feature flag for all of the 5.0.x series?

Reply:
Discussed offline: `featureFlagTimeseriesCollection` is now enabled by default on the latest server versions, so we don't need to check for this server parameter. I did bump the `minServerVersion` to `5.0`, though.